docs/deploying.md

```diff
@@ -0,0 +1,230 @@
+
+![datastore_mode](images/datastore.png)
+
+You may choose whichever region you like, however, it can only be specified once
+and cannot be undone. Google Cloud Platform currently allows only one Datastore
+database per project, so you would need to create a new project to change the
+database location.
+
+### Cloud Storage
+
+Cloud Storage is used to store large blobs that don't fit in Datastore.
+
+Create a simple bucket to hold all open saves blobs. This bucket has to be globally
+unique.
+
+```bash
+export BUCKET_PATH=gs://<your-unique-bucket-name>
+gsutil mb $BUCKET_PATH
+```
+
+### Deploying
+
+Run the following commands to deploy the containerized application to Cloud Run:
+This uses the beta version of the Cloud Run service because we are using the
+VPC connector and http2 features.
+
+```bash
+export TAG=gcr.io/triton-for-games-dev/triton-server:testing
+export SERVICE_NAME="open-saves"
+gcloud beta run deploy $SERVICE_NAME \
+                  --platform=managed \
+                  --region=$GCP_REGION \
+                  --image=$TAG \
+                  --set-env-vars="OPEN_SAVES_BUCKET="$BUCKET_PATH \
+                  --set-env-vars="OPEN_SAVES_PROJECT="$GCP_PROJECT \
+                  --set-env-vars="OPEN_SAVES_CACHE"=$REDIS_IP":"$REDIS_PORT \
+                  --allow-unauthenticated \
+                  --vpc-connector $VPC_CONNECTOR \
+                  --use-http2
+```
+
+Grab the endpoint and save it to an environment variable.
+
+```bash
+ENDPOINT=$(\
+gcloud run services list \
+  --project=$GCP_PROJECT \
+  --region=$GCP_REGION \
+  --platform=managed \
+  --format="value(status.address.url)" \
+  --filter="metadata.name="$SERVICE_NAME)
+
+ENDPOINT=${ENDPOINT#https://} && echo ${ENDPOINT}
+```
+
+Next, run the example client application to make sure everything worked.
+Make sure you have Go v1.14 or later installed.
+
+```bash
+git clone https://github.com/googleforgames/open-saves
+go run examples/grpc-all/main.go -address=$ENDPOINT:443
+```
+
+If you succesfully ran the above client code, you should see some info logged to the command line.
+The client code does the following:
+
+1. Creates a store to hold all of our data.
+2. Creates a record inside the store.
+3. Accesses the record via a `GetRecord` request
+4. Creates another record to store our blob.
+5. Uploads a blob via the stream in `CreateBlob`.
+
+## Check to see everything worked
+
+### Check Datastore
+
+In your browser, navigate to the [Datastore dashboard](https://console.cloud.google.com/datastore).
+You should see several entities here, from running the example code. Try looking for different
+kinds in the search bar at the top, specifically "Store", "Record", "Blob".
+
+### Check Memorystore
+
+While you can't see individual keys directly, navigate to the [Memorystore dashboard](https://console.cloud.google.com/memorystore)
+Select the instance that you created, and then select the "Keys in database" graph.
```

Comment:
My keys didn't show up right away. Maybe note this and the number of keys you should expect to see.

Reply:
Added clarifiation.

Also, I noted you should see "several". This example might change in the future so I don't want to fix this to a specific number of keys.